Description: Add xcb library to ANGLE X11 libs for Vulkan XCB backend
 ANGLE's libGLESv2.so fails to link due to missing xcb symbols when the
 Vulkan XCB backend is enabled. The xcb library is needed by
 WindowSurfaceVkXcb.cpp and DisplayVkXcb.cpp.
Author: Juan Manuel Méndez Rey <juan.mendezr@proton.me>
Bug-Debian: https://bugs.debian.org/915400
Forwarded: not-needed
Last-Update: 2025-11-24

diff --git a/third_party/angle/BUILD.gn b/third_party/angle/BUILD.gn
index 595e09a..8f477d7 100644
--- a/third_party/angle/BUILD.gn
+++ b/third_party/angle/BUILD.gn
@@ -706,6 +706,7 @@ angle_static_library("angle_gpu_info_util") {
         "X11",
         "Xi",
         "Xext",
+        "xcb",
       ]
     }

